adapters/eplanning/eplanning.go

```diff
@@ -232,13 +286,29 @@ func verifyImp(imp *openrtb.Imp) (*openrtb_ext.ExtImpEPlanning, error) {
 	return &impExt, nil
 }
 
+func indexOf(element string, data []string) int {
+	for k, v := range data {
+		if element == v {
+			return k
+		}
+	}
+	return -1
+}
+
+func getSizesSortedByPriority(sizeArray []openrtb.Format) []openrtb.Format {
+	sort.Sort(byPriority(sizeArray))
+	return sizeArray
```

Comment:
Since `sort.Sort` is a "destructive" call to the array, there's no need to return it. Consider inlining `sort.Sort(byPriority(sizeArray))`.